Changes

```diff
@@ -167,6 +167,10 @@ Working version
   [%extension_constructor] instead of Obj.Extension_constructor.of_val.
   (Basile Clément, review by Vincent Laviron and Nicolás Ojeda Bär)
 
+- #13589: Expose Sys.io_buffer_size, the size of internal buffers used by the
+  runtime system and the `unix` library.
+  (@EruEri and Nicolás Ojeda Bär, review by Daniel Bünzli and Nicolás Ojeda Bär)
```

Comment:
For EruEri I suggest to use the full name Yves Ndiaye (as found in the commit email address), following our usual conventions.